mobile_platform:
  # Publish all joint states
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controller
  roll_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: roll_joint

  pitch_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: pitch_joint

  yaw_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: yaw_joint

  gazebo_ros_control/pid_gains:
    roll_joint:
      p: 400.0
      i: 0.01
      d: 10.0

    pitch_joint:
      p: 400.0
      i: 0.01
      d: 10.0

    yaw_joint:
      p: 400.0
      i: 0.01
      d: 10.0
